tools/cli/app.go

```diff
@@ -79,6 +79,12 @@ func NewCliApp() *cli.App {
 			Usage:  "optional argument for transport protocol format, either 'grpc' or 'tchannel'. Defaults to tchannel if not provided",
 			EnvVar: "CADENCE_CLI_TRANSPORT_PROTOCOL",
 		},
+		cli.StringFlag{
+			Name:   FlagTLSCertPathWithAlias,
+			Value:  "",
```

Comment:
nit: delete this line to keep the style consistent

Reply:
Done, thanks